### PR TITLE
[ADEN-9528] Send comics=1 page level key-val to GAM

### PR DESCRIPTION
**Pull request by Loczi94**
## Links

https://wikia-inc.atlassian.net/browse/ADEN-9528

### CODE REVIEW

File: app/modules/ads/index.js [hunk 1]
```diff
@@ -371,6 +371,9 @@ class Ads {
   startAdEngine() {
     const { AdEngine } = window.Wikia.adEngine;
 
+    const { taxonomyService } = window.Wikia.adServices;
```
Comment (preludek) [inline app/modules/ads/index.js]: we should move this import up a line so all imports are grouped together

File: app/modules/ads/index.js [hunk 2]
```diff
@@ -371,6 +371,9 @@ class Ads {
...
+    taxonomyService.configureComicsTargeting();
```
Comment (preludek) [inline app/modules/ads/index.js]: is startAdEngine the correct place for that?

File: app/modules/ads/setup.js [hunk 3]
```diff
@@ -179,6 +179,12 @@ export const adsSetup = {
     context.set('services.moatYi.enabled', instantConfig.isGeoEnabled('wgAdDriverMoatYieldIntelligenceCountries'));
     context.set('services.nielsen.enabled', instantConfig.isGeoEnabled('wgAdDriverNielsenCountries'));
 
+    if (instantConfig.get('icTaxonomyComicsTag')) {
+      context.set('services.taxonomy.comics.enabled', instantConfig.get('icTaxonomyComicsTag'));
```
Comment (preludek) [inline app/modules/ads/setup.js]: From my understanding we later check only if this flag is true so maybe just set it to true? If instantConfig.get('icTaxonomyComicsTag') is not true we won't get here anyway.

File: app/modules/ads/index.js (hunk 1 above)
Comment (preludek) [inline app/modules/ads/index.js]: we should move this import up a line so all imports are grouped together

File: app/modules/ads/index.js (hunk 2 above)
Comment (preludek) [inline app/modules/ads/index.js]: is startAdEngine the correct place for that? Maybe move it with rest of the tracking ->  triggerPageTracking

File: app/modules/ads/setup.js (hunk 3 above)
Comment (preludek) [inline app/modules/ads/setup.js]: From my understanding we later check only if this flag is true so maybe just set it to true? If instantConfig.get('icTaxonomyComicsTag') is not true we won't get here anyway.

**Review by preludek — CHANGES_REQUESTED** [review]

File: app/modules/ads/setup.js (hunk 3 above)
Comment (Loczi94) [reply app/modules/ads/setup.js]: Yep, make sense.

**Review by preludek — APPROVED** [review]

**Pull request merged**